modules/pubmaticBidAdapter.js

```diff
@@ -347,17 +454,17 @@ export const spec = {
   isBidRequestValid: bid => {
     if (bid && bid.params) {
       if (!utils.isStr(bid.params.publisherId)) {
-        utils.logWarn(BIDDER_CODE + ' Error: publisherId is mandatory and cannot be numeric. Call to OpenBid will not be sent.');
+        utils.logWarn(BIDDER_CODE + ' Error: publisherId is mandatory and cannot be numeric. Call to OpenBid will not be sent for ad unit:');
```

Comment:
is this message incomplete? did you want to pass another argument to the function? we should tell about which adUnit we are showing the message

Reply:
Mentioned adSlot 
Done